mythril/laser/ethereum/state.py

```diff
@@ -90,19 +90,23 @@ def get_word_at(self, index: int):
 
     def __getitem__(self, item):
         if isinstance(item, slice):
+            start, step, stop = item.start, item.step, item.stop
             try:
+                if start is None:
```

Comment:
The following would be possible here too right?
```suggestion
                start = start or 0
```

Reply:
You could even make a oneliner:
```
start, step, stop = item.start or 0, item.step or 1, item.stop or self.calldatasize
```

Reply:
Or just make it three lines

Reply:
Sad